token-client-kotlin-demo/src/main/kotlin/no/nav/security/token/support/ktor/oauth/TokenResolver.kt

```diff
@@ -0,0 +1,15 @@
+package no.nav.security.token.support.ktor.oauth
+
+import no.nav.security.token.support.client.core.context.JwtBearerTokenResolver
+import no.nav.security.token.support.ktor.jwt.ClientAssertion
+import java.util.Optional
+
+class TokenResolver(
+    private val client: ClientAssertion
+) : JwtBearerTokenResolver {
+
+    // Override default client_assertion jwt, with specified Idp jwt
+    override fun token(): Optional<String> {
```

Comment:
Why `Optional` ?

Reply:
The interface implemented has a signature of `Optional<String>`